ssl/ssl_local.h

```diff
@@ -2568,6 +2570,10 @@ __owur STACK_OF(SSL_CIPHER) *ssl_get_ciphers_by_id(SSL_CONNECTION *sc);
 __owur int ssl_x509err2alert(int type);
 void ssl_sort_cipher_list(void);
 int ssl_load_ciphers(SSL_CTX *ctx);
+# ifndef OPENSSL_NO_ECH
+int ssl_cipher_list_to_bytes(SSL_CONNECTION *s, STACK_OF(SSL_CIPHER) *sk,
+                             WPACKET *pkt);
+# endif
```

Comment:
As this works just fine, please move it unconditionally.

Reply:
ack